### ISSUE TITLE
Videojs Tech is not getting duration/time from Dashjs media player

### ISSUE THREAD

**Issue by zeynelkandil** [videojs/video.js#3870]
I am trying to play a Live Dash source via videojs. 

My source is this;
http://ss01eu.boxencode.com/152/Live_trCNN1/Live_trCNN1.mpd

And it shows duration and time in DashIF player. I tried in [this player](http://dashif.org/reference/players/javascript/v2.3.0/samples/dash-if-reference-player/index.html)

But the problem is videojs player shows wrong currentTime and duration. [Wrong duration/time for MPEG-DASH live streams](https://github.com/videojs/video.js/issues/1977) says that video js tech is getting duration and current time from mediaPlayer of dashjs. But when I got deep in the code i saw that it is not getting duration value from mediaPlayer of dashjs.

The code that is expected to return dash mediaplayer duration is  this;

```
  /*
   * When using a source handler, prefer its implementation of
   * any function normally provided by the tech.
   */
  var deferrable = ['seekable', 'duration'];

  deferrable.forEach(function (fnName) {
    var originalFn = this[fnName];

    if (typeof originalFn !== 'function') {
      return;
    }

    this[fnName] = function () {
      if (this.sourceHandler_ && this.sourceHandler_[fnName]) {
        return this.sourceHandler_[fnName].apply(this.sourceHandler_, arguments);
      }
      return originalFn.apply(this, arguments);
    };
  }, _Tech.prototype);
```

When i debug this and view the content of sourceHandler the duration function of dashjs is neither in the `sourceHandler_` nor in its prototype chain. It is in `sourceHandler_.mediaPlayer_` So it is not getting the duration from dashjs mediaplayer it gets duration from its original function. And this cause videojs player to show `Number.MAX_VALUE` as duration

The same thing happens in currentTime. However i could not even see any attempt to get the currentTime from time function of dashjs mediaplayer. It directly gets the time from video element

```
Html5.prototype.currentTime = function currentTime() {
    return this.el_.currentTime;
  };
```

And this cause a weird number in currentTime.

What is the problem? I tried latest versions of videojs dash.all.debug.js and videojs-dash.js. But nothings changed. What is wrong?

**Comment by forbesjo** [comment]
This is fixed in version 2.6.1 of contrib-dash

**Comment by gkatsev** [comment]
Thanks @forbesjo.
